src/App.js

```diff
@@ -1,42 +1,37 @@
 import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import './index.css';
+import {
+  BrowserRouter,
+  Route,
+  Switch,
+} from 'react-router-dom';
+import { Container } from 'reactstrap';
 import TopNav from './components/TopNav/TopNav';
 import Home from './components/Home/Home';
-import { Container } from 'reactstrap';
 import Profile from './components/Profile/Profile';
 import TopMovieList from './components/TopMovieList/TopMovieList';
 import MoviePage from './components/MoviePage/MoviePage';
 import UpdateBar from './components/UpdateBar/UpdateBar';
 import Register from './components/Register/Register';
 import Login from './components/Login/Login';
 import Account from './components/Account/Account';
-import { connect } from 'react-redux';
-import { BrowserRouter, Route, Switch } from "react-router-dom";
-import axios from 'axios';
-import { setCurrentUser, setNewUsers, fetchList } from './redux/actions';
-
+import {
+  setCurrentUser,
+  setNewUsers,
+  fetchList,
+} from './redux/actions';
+import http from './utils/http';
+import './index.css';
 import './App.css';
 
 class App extends PureComponent {
-
   componentDidMount() {
-    axios('api/users/new-registers')
-      .then(users => {
-        this.props.setNewUsers(users.data);
+    http.users.get.newRegisters()
+      .then(({ data }) => {
+        this.props.setNewUsers(data);
       })
-      .catch(err => console.log(err));
-    //   console.log('here')
-    // axios('api/users/current')
-    //   .then(user => {
-    //     console.log('here2')
-    //     this.props.setCurrentUser(user);
-    //   })
-    //   .then(() => {
-    //     this.props.fetchList();
-    //     console.log('here3')
-    //   })
-    //   .catch(err => console.log(err));
+      .catch(console.log);
```

Comment:
`err => console.log(err)` === `console.log`

Reply:
interesting... is this common?